src/app/Main/Body/QueryPage/QueryPage-Container.component.tsx

```diff
@@ -342,6 +345,22 @@ export function QueryPageContainerComponent(): JSX.Element {
         });
     };
 
+    const handleClickQueriesDialogOpen = (): void => {
+        setQueryDialogIsOpen(true);
+    };
+
+    const handleCloseQueriesDialog = (): void => {
+        setQueryDialogIsOpen(false);
+    };
+
+    const handleApplyQuery = async (defaultQuery: string): Promise<void> => {
```

Comment:
Where is the async part here? I see no await or .then(). So why does the handler return a promise?

Reply:
Done. Was an copy&paste error